packages/abi-gen-wrappers/src/generated-wrappers/coordinator_registry.ts

```diff
@@ -219,32 +198,42 @@ export class CoordinatorRegistryContract extends BaseContract {
         ]);
         const provider = providerUtils.standardizeOrThrow(supportedProvider);
         const constructorAbi = BaseContract._lookupConstructorAbi(abi);
-        [] = BaseContract._formatABIDataItemList(
-            constructorAbi.inputs,
-            [],
-            BaseContract._bigNumberToString,
-        );
+        [] = BaseContract._formatABIDataItemList(constructorAbi.inputs, [], BaseContract._bigNumberToString);
```

Comment:
we don't run `prettier` on `abi-gen-wrappers` generated wrappers. I think we should, as long as we are comparing diffs on them.

Reply:
totally agree. i've already got Python set up to run `black` (akin to `prettier`) on wrappers.  granted, that's because i'm depending on `black` to align indents and do other whitespace changes necessary to satisfy the Python interpreter, but it sure does make for nicely legible code in the end!

Reply:
Resolved by creating issue https://github.com/0xProject/0x-monorepo/issues/1917